readme.md

```diff
@@ -89,6 +89,7 @@ $ cpy --help
 
   Example
     $ cpy 'src/*.png' dist
+    $ cpy 'src/*.png' '!src/*.ico' dist
```

Comment:
I've read that readme now so I'm not sure. `ignore` options should be used instead of negation patterns?